config/swagger.yaml

```diff
@@ -112,6 +112,71 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security:
+  /headers-by-hash:
+    get:
+      tags:
+        - external
+        - gossip
+      operationId: GetHeadersByHash
+      description: 'Get N headers from hash (down)'
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: hash
+          description: 'Hash of first block header to fetch'
+          required: true
+          type: string
+        - in: query
+          name: number
+          description: 'Number of hashes to fetch'
+          type: integer
+          default: 1
+      responses:
+        '200':
+          description: The headers found
+          schema:
+            type: array 
+            items:
+              $ref: '#/definitions/Header'
+        '400':
+          description: Invalid hash
+          schema:
+            $ref: '#/definitions/Error'
+        '401':
```

Comment:
Shall this rather be included in 400? 401 seems auth specific https://tools.ietf.org/html/rfc7235#section-3.1
So 400 would become with description "Invalid hash or number"

Reply:
>  So 400 would become with description "Invalid hash or number"
I mean description in the schema. Of course error text can stay distinct in the code.

Reply:
It doesn't really matter, does it. We are going to remove the http interface or are going to go over the swagger definitions, which are a bit broken anyway.